File: 02.Programming Fundamentals - септември 2016/02.DataTpeAndVariablesExercises/13.VowelOrDigit/VowelOrDigit.cs
using System;

class VowelOrDigit
{
    static void Main(string[] args)
    {
        int i = 0;
        string letter = Console.ReadLine();
        bool result = int.TryParse(letter, out i);
                                         
        if (letter == "A" || letter == "a" ||
            letter == "E" || letter == "E" ||
            letter == "I" || letter == "i" ||
            letter == "O" || letter == "o" ||
            letter == "U" || letter == "u" ||
            letter == "Y" || letter == "y")
        {
            Console.WriteLine("vowel");
        }
        else if (result)
        {
            Console.WriteLine("digit");
        }
        else
        {
            Console.WriteLine("other");
        }
    }
}

File: 05.Java DB Fundamentals/02.Databases Frameworks - Hibernate & Spring Data/src/_01JavaBasicExercises/VowelOrDigit.java
package _01JavaBasicExercises;

import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

/**
 * Created by stefa on 2.7.2017 г..
 */
public class VowelOrDigit {
    public static void main(String[] args) throws IOException {
        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
        String[] vowel = {"A", "E", "I", "O", "U"};
        String input = reader.readLine().toUpperCase();
        if (Character.isDigit(input.charAt(0))) {
            System.out.println("digit");
        } else {
            boolean isVowel = false;
            for (String letter : vowel) {
                if (letter.equals(input)) {
                    isVowel = true;
                }
            }
            if (isVowel) {
                System.out.println("vowel");
            } else {
                System.out.println("other");
            }
        }
    }
}
